easybuild/easyconfigs/g/GDAL/GDAL-1.11.5-foss-2015a.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '1.11.5'
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'opt': True, 'usempi': True}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('KEALib', '1.4.4'),
+    ('PostgreSQL', '9.5.3'),
+    ('netCDF', '4.3.3.1'),
+    ('libspatialite', '4.3.0a'),
+    ('libpng', '1.6.20'),
+    ('expat', '2.1.0'),
+    ('PCRE', '8.37'),        # could probably be via sqlite, but currently not.
+#    ('sqlite', )            # via libspatialite
+#    ('libxml2', '2.9.3'),   # via libspatialite
+#    ('GEOS', '3.5.0'),      # via libspatialite
+#    ('HDF5', '1.8.16'),     # via netCDF and KEALib
+#    ('zlib', '1.2.8'),      # via HDF5 and libpng
```

Comment:
maybe it makes more sense to include a comment next to libspatialite mentioning it also pulls in SQLite, libxml2 and GEOS?